constructor_io/constructor_io.py

```diff
@@ -262,3 +214,33 @@ def track_search(self, term, **kwargs):
                                    json=params,
                                    auth=(self._api_token, ""))
         return True
+
+    def _item_method(self, method_name, request_method, data, url_params=None):
+        return self._non_retrieve_method(
+            method_name=method_name, method_url="v1/item", request_method=request_method,
+            data=data, url_params=url_params
+        )
+
+    def _item_batch_method(self, method_name, request_method, data, url_params=None):
+        return self._non_retrieve_method(
+            method_name=method_name, method_url="v1/batch_items", request_method=request_method,
+            data=data, url_params=url_params
+        )
+
+    def _format_items_batch_data(self, items, autocomplete_section, **kwargs):
+        for item in items:
+            self._add_params_from_kwargs(item, **kwargs)
+        return {"items": items, "autocomplete_section": autocomplete_section}
+
+    def _non_retrieve_method(self, method_name, method_url, request_method, data,
```

Comment:
What would be a non retrieve method? A method that is not `GET`?

Reply:
added docstring in https://github.com/Constructor-io/constructorio-python/pull/9/commits/d3315a56689024b7ce6e60ceeb97b2d15f1a7a9c